src/main/diagnostics/descriptorActionProvider.ts

```diff
@@ -31,7 +31,60 @@ interface DirectDependencyIssue {
  * 2. Adds severity icon to the descriptor file in the places were the infected dependency exists
  */
-export class DescriptorActionProvider extends AbstractFileActionProvider {
+export class DescriptorActionProvider extends AbstractFileActionProvider implements vscode.CodeActionProvider {
+    private _processedMap: Map<vscode.Uri, Map<string, DirectDependencyInfo>> = new Map<vscode.Uri, Map<string, DirectDependencyInfo>>();
+
+    /** @Override */
+    public activate(context: vscode.ExtensionContext) {
+        super.activate(context);
+        context.subscriptions.push(
+            this,
+            vscode.languages.registerCodeActionsProvider(DescriptorUtils.DESCRIPTOR_SELECTOR, this, {
+                providedCodeActionKinds: [vscode.CodeActionKind.Empty]
+            } as vscode.CodeActionProviderMetadata)
+        );
+    }
+
+    /** @Override */
+    provideCodeActions(document: vscode.TextDocument, range: vscode.Range | vscode.Selection): vscode.Command[] | undefined {
+        // Search if the file had issues in the scan
+        const fileNode: FileTreeNode | undefined = this._treesManager.issuesTreeDataProvider.getFileIssuesTree(document.uri.fsPath);
+        if (fileNode instanceof DescriptorTreeNode) {
+            // Get the dependency information we created in the specific range in the document
+            let processedDependencies: Map<string, DirectDependencyInfo> | undefined = this._processedMap.get(document.uri);
+            if (!processedDependencies) {
+                return undefined;
+            }
+            let rangeDependencies: DirectDependencyInfo[] = Array.from(processedDependencies.values()).filter(dependency =>
+                dependency.range.some(pos => range.intersection(pos))
+            );
+            let infectedDependencies: Set<DependencyIssuesTreeNode> = new Set<DependencyIssuesTreeNode>();
+            // Get all the infected dependencies for a direct dependency as a set
+            for (const directDependencyInfo of rangeDependencies) {
+                for (const issueInfo of directDependencyInfo.diagnosticIssues.values()) {
+                    for (const infectedDependencyId of issueInfo.infectedDependencies) {
+                        let infectedDependency: DependencyIssuesTreeNode | undefined = fileNode.getDependencyByID(infectedDependencyId);
+                        if (infectedDependency) {
+                            infectedDependencies.add(infectedDependency);
+                        }
+                    }
+                }
+            }
+            let commands: vscode.Command[] = [];
+            // Add jump to tree commands for each infected dependency
+            for (const infectedDependency of infectedDependencies) {
+                commands.push({
+                    command: 'jfrog.issues.select.node',
+                    title: 'Show infected dependency ' + infectedDependency.componentId + ' in issues tree',
```

Comment:
Wrap the infected dependency id with quotes
```suggestion
                    title: "Show infected dependency '" + infectedDependency.componentId + "' in issues tree",
```